stdlib/math/math.flux

```diff
@@ -217,837 +317,1243 @@ builtin atan : (x: float) => float
 // math.atan2(y: +Inf, x:x)      // Returns +Pi/2
 // math.atan2(y: -Inf, x:x)      // Returns -Pi/2
 // ```
+//
 builtin atan2 : (y: float, x: float) => float
 
-// atanh is a function that returns the inverse hyperbolic tangent of x.
+// atanh returns the inverse hyperbolic tangent of `x`.
 //
 // ## Parameters
-// - x: is the value used in the operation.
+// - x: Value to operate on.
 //
-//   x should be greater than -1 and less than 1, otherwise the operation
-//   will return NaN.
+//   `x` should be greater than -1 and less than 1. Otherwise the operation
+//   will return `NaN`.
 //
-// ## Example
+// ## Examples
 //
-// ```
+// ### Return the hyperbolic tangent of a value
+// ```no_run
 // import "math"
 //
-// math.atanh(x: 0.22)
+// math.atanh(x: 0.22) // 0.22365610902183242
 // ```
 //
-// ## Special Cases
-//
+// ### Use math.atanh in map
 // ```
 // import "math"
+// # import "sampledata"
+// #
+// # data = sampledata.float() |> map(fn: (r) => ({r with _value: r._value * .01}))
+//
+// < data
+// >     |> map(fn: (r) => ({r with _value: math.atanh(x: r._value)}))
+// ```
+//
+// ## Special cases
 //
+// ```no_run
 // math.atanh(x: 1)   // Returns +Inf
 // math.atanh(x: ±0)  // Returns ±0
 // math.atanh(x: -1)  // Returns -Inf
 // math.atanh(x: <-1) // Returns NaN
 // math.atanh(x: >1)  // Returns NaN
 // math.atanh(x: NaN) // Returns NaN
 // ```
+//
 builtin atanh : (x: float) => float
 
-// cbrt is a function that returns the cube root of x.
+// cbrt returns the cube root of x.
 //
 // ## Parameters
-// - x: is the value used in the operation.
+// - x: Value to operate on.
 //
-// ## Example
+// ## Examples
 //
-// ```
+// ### Return the cube root of a value
+// ```no_run
 // import "math"
 //
-// math.cbrt(x: 1728.0)
+// math.cbrt(x: 1728.0) // 12.0
 // ```
 //
-// ## Special Cases
-//
+// ### Use math.cbrt in map
 // ```
 // import "math"
+// import "sampledata"
+//
+// < sampledata.float()
+// >     |> map(fn: (r) => ({r with _value: math.cbrt(x: r._value)}))
+// ```
+//
+// ## Special cases
 //
+// ```no_run
 // math.cbrt(±0)   // Returns ±0
```

Comment:
Wow, acknowledging that there is both + and - zero in special cases. Nice to see this behavior described so thoroughly!